.ebextensions/nginx.config

```diff
@@ -0,0 +1,232 @@
+files:
```

Comment:
The contents of the files in `.ebextensions/nginx.config` are derived from the _excellent_ [h5bp/server-configs-nginx](https://github.com/h5bp/server-configs-nginx). That crew (which includes some Google folk) spends a lot of time researching and tuning these configs for maximum performance.

Reply:
Oh yeah, and Elastic Beanstalk will pick up any `.config` files in `.ebextensions` and churn through them during the deploy process.

The ever-so-closely-named `*.conf` files are Nginx configuration files.